app/controllers/todos_controller.rb

```diff
@@ -8,10 +8,12 @@ def index
     )
 
     if todos.valid?
-      render json: todos.result
+      render :json => todos.result
```

Comment:
what the differences and why have you changed it?

Reply:
There is no difference. I just like to write in such manner :)

Reply:
okay